airflow/www/app.py

```diff
@@ -154,6 +156,9 @@ def integrate_plugins():
                                         href=ml["href"],
                                         category=ml["category"],
                                         category_icon=ml["category_icon"])
+                for bp in sorted(flask_blueprints, key=lambda x: x["name"]):
```

Comment:
Just curious, why is this sorting required?

Reply:
I see. Hm, but only menu links are sorted, which makes sense since they are added to then menu so the order should be deterministic. But I think for blueprints (like views) that's not required. But maybe I'm wrong...

Reply:
Oh you are right! I shall get rid of it when i’m next at laptop.